definitions/npm/formik_v2.x.x/flow_v0.69.0-v0.103.x/formik_v2.x.x.js

```diff
@@ -79,6 +79,7 @@ declare module 'formik/@flow-typed' {
     onReset?: (values: Values, formikHelpers: FormikHelpers<Values>) => void,
     validationSchema?: (() => Schema) | Schema,
     validate?: (values: Values) => void | {} | Promise<FormikErrors<Values>>,
```

Comment:
I know that it's not your changes but could you add the possibility to using generic type for values